docs/program/apis/cloud.md

```diff
@@ -0,0 +1,206 @@
+from viam.rpc.dial import DialOptions, Credentials
+from viam.app.viam_client import ViamClient
+
+async def connect() -> ViamClient:
+    dial_options = DialOptions(
+        auth_entity='mrroboto.this_is_just_an_example.viam.cloud',  # The URL of your robot.
+        credentials=Credentials(
+            type='robot-location-secret',
+            payload='YOUR ROBOT LOCATION SECRET' # The location secret of your robot
+        )
+    )
+    return await ViamClient.create_from_dial_options(dial_options)
+
+async def main():
+
+  # Make a ViamClient
+  viam_client = await connect()
+  # Instantiate an AppClient called "cloud" to run the cloud app API methods on
+  cloud = await viam_client.app_client()
+```
+
+Then, run methods on the `AppClient` according to the code examples for each method below.
+
+{{% /alert %}}
+
+The cloud API supports the following methods:
+
+{{< readfile "/static/include/services/apis/cloud.md" >}}
+
+### ListOrganizations
+
+List the organizations the user is an authorized user of.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- None
+
+**Returns:**
+
+- [(List[viam.proto.app.Organization])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Organization): A list of the organization or organizations of which the user is an authorized owner.
+
+```python {class="line-numbers linkable-line-numbers"}
+org_list = await cloud.list_organizations()
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.list_organizations).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### ListOrganizationMembers
+
+List the members and invites of the organization that you are currently authenticated to.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- None
+
+**Returns:**
+
+- [(Tuple[List[viam.proto.app.OrganizationMember]])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.OrganizationMember): A tuple containing to lists:
+  - The first (`[0]`) is a list of organization members.
+  - The second (`[1]`) is a list of organization invites.
+
+```python {class="line-numbers linkable-line-numbers"}
+member_list = await cloud.list_organization_members()
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.list_organization_members).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetOrganizationNamespaceAvailability
+
+Check the availability of an organization namespace.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `public_namespace` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The organization namespace to check.
+Namespaces can only contain lowercase alphanumeric and dash characters.
+
+**Raises:**
+
+- GRPCError: An error is raised if an invalid namespace (for example, `""`) is provided.
```

Comment:
Tense on this is a little weird and I think policy as-is to not have examples in commas but could be mistaken
```suggestion
- GRPCError: The error raised if an invalid namespace has been provided.
```
Then put example at the top bc this seems like method function right?

Reply:
This came directly from the python SDK docs. I don't think I understand.

Reply:
I think we should change it to speak to the user is all! I'm just suggesting this change in tense, as I think the way that this was written was ok for the Python SDK docs but not ok for docs.viam.com. What's being returned is the actual error, as far as I understand, so I think our docs should reflect that.

Reply:
I guess I don't actually know the whole error message--I guess GRPCError should be put in backticks if it is indeed the literal error?

Reply:
```suggestion
- `GRPCError`: This error is raised if an invalid namespace (for example, `""`) is provided.
```
Does this make sense?

Reply:
yep that looks great!